app/_hub/kong-inc/proxy-cache-advanced/_index.md

```diff
@@ -57,7 +57,9 @@ params:
         - application/json
       datatype: array of string elements
       description: |
-        Upstream response content types considered cacheable. The plugin performs an **exact match** against each specified value; for example, if the upstream is expected to respond with a `application/json; charset=utf-8` content-type, the plugin configuration must contain said value or a `Bypass` cache status is returned.
+        Upstream response content types considered cacheable.
+        The wildcard content type can be used to match any specific type or subtype.
+        For example, `application/*` matches any subtypes of the application, `*/*` matches all content types. Note that `*/*` does not match with a content type that contains a parameter. If the response content type does not match the configuration, a `Bypass` cache status is returned.
```

Comment:
@vm-001 Can I provide `application/json;*` to match `application/json; charset=utf-8` _and_ `application/json`?

Reply:
No, you can't. `application/json;*`  isn't a valid content type.

Reply:
How about `application/json; charset=*`?

Our matching logic requires an exact match for the header. It'd be nice to say `application/json`, whether `charset=utf-8` is provided or not

Reply:
The matching behavior of Proxy Cache plugin now follows the Request Validator. I suggest let's keep the consistency with other plugins. Then, we can figure out the best match behavior and apply it to all plugins at once at the next stage. Do you agree with me?